tests/012-test-container-root-lv-options.sh

```diff
@@ -0,0 +1,74 @@
+source $SRCDIR/libtest.sh
+
+# Test CONTAINER_ROOT_LV_NAME, CONTAINER_ROOT_LV_MOUNT_PATH and CONTAINER_ROOT_LV_SIZE directives.
+# Returns 0 on success and 1 on failure.
+test_container_root_volume() {
+  local devs=$TEST_DEVS
+  local test_status=1
+  local testname=`basename "$0"`
+  local vg_name="dss-test-foo"
+  local container_root_lv_name="container-root-lv"
+  local container_root_lv_mount_path="/var/lib/containers"
```

Comment:
I think these are local variables and we can use shorter names. Say.

`root_lv_name` and `root_lv_mount_path`